logconfig/manager.go

```diff
@@ -450,137 +450,141 @@ func InitLogManager(config *maestroConfig.YAMLMaestroConfig) (err error) {
+		greasego.AddSink(unixSockSink)
 
-	debugging.DEBUG_OUT("targets:", len(config.Targets))
-	for n := 0; n < len(config.Targets); n++ {
-		if len(config.Targets[n].File) > 0 { // honor any substitution vars for the File targets
-			config.Targets[n].File = maestroConfig.GetInterpolatedConfigString(config.Targets[n].File)
+		syslogSock := config.GetSyslogSocket()
+		if len(syslogSock) > 0 {
+			syslogSink := greasego.NewGreaseLibSink(greasego.GREASE_LIB_SINK_SYSLOGDGRAM, &syslogSock)
+			greasego.AddSink(syslogSink)
 		}
-		opts := greasego.NewGreaseLibTargetOpts()
-		greasego.AssignFromStruct(opts, config.Targets[n]) //, reflect.TypeOf(config.Targets[n]))
 
-		if config.Targets[n].Flag_json_escape_strings {
-			greasego.TargetOptsSetFlags(opts, greasego.GREASE_JSON_ESCAPE_STRINGS)
+		// First, setup the internal maestro logging system to deal with toCloud target
+		// This requires creating the default Symphony client:
+		var symphonyClient *wwrmi.Client
+		var symphonyErr error
+		if config.Symphony != nil {
+			symphonyClient, symphonyErr = wwrmi.GetMainClient(config.Symphony)
+		} else {
+			log.MaestroDebugf("Symphony / RMI API server not configured.\n")
 		}
 
-		debugging.DEBUG_OUT("%+v\n", opts.FileOpts)
-		debugging.DEBUG_OUT("%+v\n", opts)
-		debugging.DEBUG_OUT("%+v\n", *opts.Format_time)
+		debugging.DEBUG_OUT("targets:", len(config.Targets))
+		for n := 0; n < len(config.Targets); n++ {
+			if len(config.Targets[n].File) > 0 { // honor any substitution vars for the File targets
+				config.Targets[n].File = maestroConfig.GetInterpolatedConfigString(config.Targets[n].File)
+			}
+			opts := greasego.NewGreaseLibTargetOpts()
+			greasego.AssignFromStruct(opts, config.Targets[n]) //, reflect.TypeOf(config.Targets[n]))
 
-		if strings.Compare(config.Targets[n].Name, "toCloud") == 0 {
-			log.MaestroInfof("\nFound toCloud target-------->\n")
-			opts.NumBanks = defaults.NUMBER_BANKS_WEBLOG
-			//			DEBUG(_count := 0)
-			if config.Symphony != nil && symphonyClient != nil && symphonyErr == nil {
-				opts.TargetCB = wwrmi.TargetCB
-			} else {
-				log.MaestroInfof("Log: 'toCloud' target is enabled, but Symphony API is not configured. Will not work.")
-				// skip this target
-				continue
+			if config.Targets[n].Flag_json_escape_strings {
+				greasego.TargetOptsSetFlags(opts, greasego.GREASE_JSON_ESCAPE_STRINGS)
 			}
 
-			// func(err *greasego.GreaseError, data *greasego.TargetCallbackData){
-			// 	DEBUG(_count++)
-			// 	debugging.DEBUG_OUT("}}}}}}}}}}}} TargetCB_count called %d times\n",_count);
-			// 	if(err != nil) {
-			// 		fmt.Printf("ERROR in toCloud target CB %s\n", err.Str)
-			// 	} else {
-			// 		buf := data.GetBufferAsSlice()
-			// 		DEBUG(s := string(buf))
-			// 		debugging.DEBUG_OUT("CALLBACK %+v ---->%s<----\n\n",data,s);
-			// 		client.SubmitLogs(data,buf)
-			// 	}
-			// }
-		}
+			debugging.DEBUG_OUT("%+v\n", opts.FileOpts)
```

Comment:
Will these debug statements need to be removed? Looks like they were originally commented out?

Reply:
I think this was originally on line 506, which seems to still be active.  I think the maestro DEBUG flag ulitimately controls whether this is enabled?  not sure.

Reply:
Yes, I think DEBUG flag controls if these statements are enabled.